### DISCUSSION TITLE
Vehicles cargo and Chinook texture

### DISCUSSION THREAD

**Discussion by STyx2909** [Vdauphin/HeartsAndMinds#1477] [General]
Greetings to all.

Hearts and Minds is among my most favorite game MODs. Probably #1.

I am still working on adding an immersion factor to the game; mostly by creating ambient animations into the main base (Altis), so you don't feel like you are alone with a bunch of mates to help the citizens and defeat Oplitas.

Aside this, there are a few points I would like some help with:
- Vehicles cargo is changed in game and very different from one in Eden editor. I suppose this is a workaround for the ViV feature, right? Otherwise, I would really like to be able to use the original settings here. (Haven't been able to find where in the code it is hidden).
- Replacing the CH47I with an unarmed version was a piece of cake. I consider respawnable vehicles have to be unarmed. But there are no way I can get the green version. LOL. Any help?

And sharing a night version of the base:
![Arma3_x64 2023-01-19 23-52-01](https://user-images.githubusercontent.com/76572473/214279508-5f2ff8ca-c616-48cc-9ecc-f867d7bbe163.png)

And a recent addition, with IDAP:
![Arma3_x64 2023-01-19 23-52-38](https://user-images.githubusercontent.com/76572473/214279595-f1fe7154-dd4a-4729-9df7-032e7c7fa3a6.png)

**Comment by Vdauphin** [comment]
Hello,
> I suppose this is a workaround for the ViV feature, right?

No, it is because you can't load container in truck vehicles.
https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/eh/server.sqf#L131-L141

> Any help?

No idea, ask in A3 Official Discord

Cheers

**Reply by STyx2909** [reply]
I reported the Chinook as a bug to the feedback system. It doesn't work at all. Weird it was never catched up before.

**Reply by TheLukio** [reply]
I've just stumbled over this as well. Placing unarmed CH-67 and then in selecting the green texture from Eden looks correct in editor, but in the misison it gets reverted to the black version. 

Even when using a vehicle init it does this, so IDK what is going on.

**Reply by STyx2909** [reply]
It is an Arma3 bug. Will be fixed in next release. Already fixed in dev. version, I checked.

**Comment by TheLukio** [comment]
Can confirm this issue has been resolved with version 2.12. All is well.
